.scala-steward.conf

```diff
@@ -1,6 +1,8 @@
 updates.ignore = [
   { groupId = "org.scalameta" },
-  { groupId = "org.webjars" }
+  { groupId = "org.webjars" },
+  { groupId = "org.webjars.npm"},
+  { groupId = "software.amazon.awssdk" }
```

Comment:
one thing we could do is:

```
dependencyOverrides = [
  {
    dependency = { groupId = "software.amazon.awssdk" },
    pullRequests = { frequency = "14 day" }
  }
]
```

Instead of disabling it completely, just reducing the frequency is probably useful

Details about the config: https://github.com/scala-steward-org/scala-steward/blob/main/docs/repo-specific-configuration.md

Reply:
Tbh I'm not sure it's even worth it : the signature algorithm ain't gonna change and that's the only thing we're really after, and it's a dependency we only use in tests. Even if it changes less often, I'll find it annoying to use resources to build version bumps for it when the bump would provide literally zero value ...